File: java-playground/src/test/java/org/gonza/javaplayground/model/MoneyTest.java
package org.gonza.javaplayground.model;

import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class MoneyTest {

    @Test
    @DisplayName("금액을 생성할 수 있다")
    void 금액_생성_테스트() {
        int 금액 = 1000;

        Money money = new Money(금액);

        assertEquals(금액, money.금액_가져오기());
    }

}

File: kotlin-playground/src/test/kotlin/org/gonza/kotlinplayground/domain/money/MoneyTest.kt
package org.gonza.kotlinplayground.domain.money

import org.junit.jupiter.api.Assertions.*
import org.junit.jupiter.api.Test

class MoneyTest {
    @Test
    fun `구입한 로또 가격을 입력하면 구입된 로또 티켓의 개수를 알 수 있다`() {
        val money = Money(14000)

        val result = money.getPaidTicketCount()

        val expected = 14
        assertEquals(expected, result)
    }
}
